drivers/clock_control/clock_stm32f1.c

```diff
@@ -23,6 +23,11 @@
 
 #if defined(STM32_PLL_ENABLED)
 
+uint32_t get_pllout_frequency(void)
+{
```

Comment:
A comment would be appreciated.
```suggestion
{
    /* Stub implementation for compatibility with clock_stm32_ll_common.
     * The PLL domain clock is only used for MCO configuration, but the
     * MCO driver never queries the PLL output clock frequency.
     */
```